buildenv/jenkins/common/build.groovy

```diff
@@ -326,8 +326,64 @@ def archive_sdk() {
     }
 }
 
+/*
+* When crash occurs in java, there will be an output in the log: IEATDUMP success for DSN='JENKINS.JVM.JENKIN*.*.*.X&DS'.
+* The X&DS is a macro that is passed to the IEATDUMP service on z/OS.
+* The dump is split into multiple files by the macro if the address space is large.
+* Each dataset is named in a similar way but there is a distinct suffix to specify the order. 
+* For example, JVM.*.*.*.001, JVM.*.*.*.002, and JVM.*.*.*.003.
+* The function fetchFile moves each dataset into the current directory of the machine, and appends the data to the core.xxx.dmp at the same time.
+* The file core.xxx.dmp is uploaded to artifactory for triage.
+*/
+def fetchFile(src, dest) {
+    // remove the user name from the dataset name
+    def offset = USER.length() + 1
+    def fileToMove = src.substring(src.indexOf("${USER}.") + offset)
+    // move a dataset to the current directory
+    sh "mv '//${fileToMove}' ${src}"
+    // append the dataset to core.xxx.dmp file
+    sh "cat ${src} >> ${dest}"
+    // remove the dataset from the current directory
+    sh "rm ${src}"
```

Comment:
Could this be contracted to avoid copying these (large) files twice?
```
sh "cat '//${fileToMove}' >> ${dest}"
sh "rm '//${fileToMove}'"
```

Reply:
Hey @keithc-ca! Sorry for late replying! I believe we don't support concatenation directly using TSO, so I first move each file to the current directory then append it to the core file. Thanks!

Reply:
> Yes, `tso delete` seems appropriate.

Hey @keithc-ca! It was tested in the build 68. Thanks!
https://hyc-runtimes-jenkins.swg-devops.com/job/Build_JDK11_s390x_zos_Jenny/68/